docs/chapter_array_and_linkedlist/linked_list.md

```diff
@@ -369,7 +376,27 @@
 === "Rust"
 
     ```rust title="linked_list.rs"
+    use std::rc::Rc;
+    use std::cell::RefCell;
+
+    #[derive(Debug)]
+    struct ListNode {
```

Comment:
Remove this class. Please refer to other languages.

Reply:
Done. Please check. :)